bin/GameIndex.dbf

```diff
@@ -16921,6 +17052,23 @@ Region = PAL-E
 // Reads data from DDS1.
 MemCardFilter = SLES-54555/SLES-53458
 ---------------------------------------------
+Serial = SLES-54559
+Name   = Free Running
+Region = PAL-E
+[patches = d6a0a3ef]
+	author=Prafull
+	// Fix character falling through floor
+	// Fixed by rearranging COP2 instructions
```

Comment:
I am okay with this description but the main issue is kind of overlooked here, the roo tof this issue is the game need a tough syncro between the COP2 and the VU0. Something PCSX2 struggle at.